docs/source/user-guide.md

```diff
@@ -327,6 +331,78 @@ model_xai = xai.insert_xai(
 # ***** Downstream task: user's code that infers model_xai and picks 'saliency_map' output *****
 ```
 
+## Saving saliency maps
+
+You can easily save saliency maps with flexible naming options, including prefix, suffix, and postfix. Additionally, you can include the confidence score for each class in the saved saliency map's name.
+
+```python
+import cv2
+import numpy as np
+import openvino.runtime as ov
+from openvino.runtime.utils.data_helpers.wrappers import OVDict
+import openvino_xai as xai
+
+def preprocess_fn(image: np.ndarray) -> np.ndarray:
+    """Preprocess the input image."""
+    resized_image = cv2.resize(src=image, dsize=(224, 224))
+    expanded_image = np.expand_dims(resized_image, 0)
+    return expanded_image
+
+def postprocess_fn(output: OVDict):
+    """Postprocess the model output."""
+    return output["logits"]
```

Comment:
```suggestion
from from typing import Mapping
import openvino_xai as xai

def preprocess_fn(image: np.ndarray) -> np.ndarray:
    """Preprocess the input image."""
    resized_image = cv2.resize(src=image, dsize=(224, 224))
    expanded_image = np.expand_dims(resized_image, 0)
    return expanded_image

def postprocess_fn(output: Mapping):
    """Postprocess the model output."""
    return output["logits"]
```

Reply:
Could you possibly replace the verbose `OVDict` import by `Mapping` in other code or docs?